Makefile

```diff
@@ -213,8 +214,10 @@ zkevm_london.bin: ${ZKEVM_MODULES_LONDON}
 	${GO_CORSET_COMPILE} -o $@ ${ZKEVM_MODULES_LONDON}
 	@$(call warn_lispX)
 
+ #This is not a typo:
+ # only a column name change between Paris and London n BLOCK_DATA that blocks us to have a conflation with London and Paris blocks
 zkevm_paris.bin: ${ZKEVM_MODULES_PARIS}
```

Comment:
This does look odd having `zkevm_paris.bin: ${ZKEVM_MODULES_PARIS}`.  Perhaps just remove `ZKEVM_MODULES_PARIS` altogether?